--- settings.from.py	2026-09-24 17:04:48.075308233 +0200
+++ settings.to.py	2026-09-24 17:31:45.939495640 +0200
@@ -654,3 +654,6 @@
 REDIRECT_LOGIN_TO_SSO = config("REDIRECT_LOGIN_TO_SSO", default=False, cast=bool)
 
 SESSION_COOKIE_AGE = config("SESSION_COOKIE_AGE", default=60 * 60 * 24 * 14, cast=int)
+
+SOCIALACCOUNT_ADAPTER = config("SOCIALACCOUNT_ADAPTER", default="users.habitat_social_account_adapter.HabitatSocialAccountAdapter")
+OIDC_ADMIN_GROUPS = config("OIDC_ADMIN_GROUPS", default="[]", cast=json.loads)
